### DISCUSSION TITLE
Random seed for RandomLinkSplit?

### DISCUSSION THREAD

**Discussion by scottshufe** [pyg-team/pytorch_geometric#3607] [Q&A] [answered]
Hi, PyG Team. Thanks for your excellent job first.

I am using the RandomLinkSplit transformation to do some link prediction research, but I find that the parameters of this class don't contain a _random_state_ option. So my question is how to set the random seed for RandomLinkSplit to obtain a fixed splitting? 

Look forward to your reply. And thanks in advance :)

**Comment by rusty1s** [comment, marked as answer]
We follow PyTorch convention here, in which you set a global seed at the top of your script, *e.g.*:
```python
from torch_geometric import seed_everything

seed_everything(42)
```
Hope this helps!

**Reply by scottshufe** [reply]
Thanks for your help! It works well for me.
